### ISSUE TITLE
Mailgun seems to fail on wiki.openzim.org

### ISSUE THREAD

**Issue by kelson42** [openzim/mediawiki-docker#15]
I have configured mailgun but if I send an email from "user:Kelson", then I get a PHP error.

**Comment by florentk** [comment]
@kelson42  I had no put API key in config file, now it's done in the tarball. But is not an env variable of the container (for moment).

**Comment by kelson42** [comment]
I had done that already. Still the same error in https://wiki.openzim.org/wiki/Special:EmailUser

**Comment by kelson42** [comment]
By activating the debug I see the error:
```
[exception] [6fa1214c5ebe6a71f9883d25] /wiki/Special:EmailUser Error from line 46 of /var/www/html/w/extensions/Mailgun/MailgunHooks.php: Class 'Http\Adapter\Guzzle6\Client' not found
```

I think this is the same pb as, probably the extension is not fully installed:
https://phabricator.wikimedia.org/T144143

**Comment by florentk** [comment]
fix in 146b862

**Comment by kelson42** [comment]
@florentk thx, seems to work fine :)
